tests/Writer/XLSX/Helper/DateIntervalHelperTest.php

```diff
@@ -41,6 +41,14 @@ public static function provideDateIntervalToExcelCases(): array
             [DateInterval::createFromDateString('1 second'), 1 / 24 / 60 / 60],
             [DateInterval::createFromDateString('2 seconds'), 2 / 24 / 60 / 60],
             [new DateInterval('PT12H30M0S'), 0.5 + 30 / 24 / 60],
+            [self::invert(new DateInterval('PT12H0M0S')), -0.5],
         ];
     }
+
+    private static function invert(DateInterval $interval): DateInterval
```

Comment:
```suggestion
    private function invert(DateInterval $interval): DateInterval
```

Reply:
Hmm, I don't think we can call a non-static method inside a static one, can we?

Reply:
That's the reason for https://github.com/openspout/openspout/pull/188#discussion_r1365225442

Reply:
But `provideDateIntervalToExcelCases()` is static, which is calling `invert()` !?

Reply:
Oh sorry, didn't see the origin.

Ok for the static then